### DISCUSSION TITLE
File Management Tab

### DISCUSSION THREAD

**Discussion by Nicholson27** [forkineye/ESPixelStick#581] [Q&A] [answered]
Hello,

New to this and not sure if I am missing something….I flashed my quinled-ESP32-AE+ with the version 4.0 beta 4. I have noticed a lot of videos showing how the SD cards have a file management tab in ESPixelStick. I do not see this even after changing the port from GIO 4 to 5. I have tried flashing with other versions as well and still no file management tab. Any ideas why it is missing?

thanks

**Comment by MartinMueller2003** [comment]
Beta 4 does not support any of the quinled platforms. You need to use a build from the current mainline

**Reply by Nicholson27** [reply]
Would this be in the ESPixelStick CI?

**Comment by MartinMueller2003** [comment, marked as answer]
Latest installer with latest code can always be found here: https://drive.google.com/file/d/1DG3wz8lbYNc371OIbeOqkzMaY5Zd2SKA/view?usp=sharing

**Reply by Nicholson27** [reply]
Thank you so much for the help....I noticed that a lot of others were using this version, but I could not find where to get it! Also, I noticed an error during flashing where my SD card was not mounting. I was able to fix this by changing the format to FAT32. Everything is working correctly now!

**Reply by Atlas-Night-Out** [reply]
how did you flash the QuinLED-ESP32-AE+ has the video I was following is using a mac and I'm on a pc? also what did you use to format fat32 as I used rufus but not fully sure if this will be ok on this setup? Also the links don't work any more!

**Comment by MartinMueller2003** [comment]
We have moved to a new mechanism for flashing that is platform independent. Use your browser to access:

https://espixelstickwebflasher.from-ct.com:5000
